src/components/Home.js

```diff
@@ -0,0 +1,59 @@
+import { useSelector } from 'react-redux';
+import CountUp from 'react-countup';
+import { FaSearchLocation } from 'react-icons/fa';
+import { FcGlobe } from 'react-icons/fc';
+import Spinner from './Spinner/Spinner';
+import ContinentList from './ContinentList';
+
+const Home = () => {
+  const {
+    continents,
+    totalData: { totalCases, totalDeaths },
+  } = useSelector((state) => state.covid);
+
+  return (
+    <>
+      {!continents.length && <Spinner />}
+      {continents.length > 0 && (
+        <>
+          <div className="globeDetails">
+            <FcGlobe className="globe" />
+            <div className="count">
+              <CountUp
+                delay={1}
+                end={totalCases}
+                separator=","
+                prefix="Total Cases: "
+                duration={3}
+              />
+              <CountUp
+                delay={1}
+                prefix="Total Deaths: "
+                end={totalDeaths}
+                separator=","
+                duration={3}
+              />
+            </div>
+          </div>
+          <form className="form">
+            <div className="search-bar">
+              <FaSearchLocation />
+              <div>
+                <input
+                  className="input-area"
+                  type="text"
+                  value={null}
+                  placeholder="Search Here"
+                  onChange={null}
+                />
```

Comment:
- [ ] The previous reviewer asked that you implement the stateless filter component. I can see that you have an input search field but `onChange` it is `null`. What you should do is that `onChange`, you want to filter the array that you fetch from the redux store to find all items whose names are included in the text you type in the search field. You can read this [article](https://dev.to/salehmubashar/search-bar-in-react-js-545l#:~:text=Creating%20the%20Search%20Bar&text=In%20your%20app.,%22%20import%20%22.%2FApp.) to have an idea of how to implement it or you can watch this [video](https://www.youtube.com/watch?v=x7niho285qs). 👍

Reply:
Implemented filter

Reply:
:+1: Fixed :heavy_check_mark: